.circleci/config.yml

```diff
@@ -60,7 +60,7 @@ jobs:
       - run:
           name: Build
           command: |
-            yarn build
+            ./ci-build.sh
```

Comment:
We could maybe simplify this by inlining the CI-specific check here:

```
if [[ -z "$ALGOLIA_API_KEY" || -z "$ALGOLIA_INDEX_NAME" ]]
then
  printf 'ERROR: %s\n' 'Algolia API key or index name not set.' >&2
  exit 1
fi
```

And we could continue to use `yarn build` (more below).

(Two subtle changes made to this snippet: [`[[` in place of `[`](http://mywiki.wooledge.org/BashFAQ/031) and single quotes to avoid the escapes.)

Reply:
Inlined in `config.yml` in 2d8c946